site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,9 +6,24 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
 ---
 
-Extensions are small software programs that customize the browsing experience. They enable users to
-tailor Chrome functionality and behavior to individual needs or preferences. They are built on web
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
+
+
+## About extensions
+
+Extensions are small software programs that customize the browsing experience. They let users
+tailor Chrome functionality and behavior in many ways, providing things like:
+
+* Games
```

Comment:
I'm very hesitant about having "games" be the first example of what extensions can do. Honestly I'm not even sure if we should include it in this list. 

Broadly speaking extensions are meant as a way to extend your web browser's functionality. You might want to customize how a feature works (e.g. replace the browser's bookmark manager), add new functionality (e.g. look up a word by highlighting it), integrate tools that keep you on track (e.g. blocking distracting websites), and so on.

While there are some games in the Chrome Web Store, I don't think this is the right focus for the extensions platform. Generally speaking I think games are better as web sites rather tightly integrated into the browser experience.

Reply:
Could we say "Delightful experiences" aka something like "Fun", instead of "Games"?

This would describe to me [this extension](https://chrome.google.com/webstore/detail/tabby-cat/mefhakmgclhhfbdadeojlkbllmecialg?hl=en).

Reply:
Changed this to "Fun & games" as the last bullet.